content/learning-how-to-teach-coders/quizzes/_index.md

```diff
@@ -2,4 +2,45 @@
 _db_id: 649
```

Comment:
Where did you get the db_id? This is generated when tilde consumes the syllabus and should probably be removed.